### ISSUE TITLE
Remote assistance not working after reboot (Again?)

### ISSUE THREAD

**Issue by lorenzocanalelc** [Ansuel/tch-nginx-gui#1044]
NB: Before submitting an issue, check if there is one already open that suits the problem you are having!
Same as #299 but that issue is now close so I decided to open up a new one

Device Model/Firmware: Tg789vac V2 Firmware UNO 17.2.0278
GUI Version:  9.6.66-e8758d63

[DebugHelper2020-12-19-1340.tar.gz](https://github.com/Ansuel/tch-nginx-gui/files/5719488/DebugHelper2020-12-19-1340.tar.gz)

(Also just saying, wouldn't it better to avoid posting the ifconfig and routing since they contain sensible datas (IP address specifically)

Description of problem, HOW TO REPRODUCE, Media/Photos: After a reboot the remote assistance doesn't work until giving at least a "curl localhost" see #299

**Comment by FrancYescO** [comment]
i already checked about this thing in the past, and probably this can happen totally random if during the startup nginx is still not ready to response requests as the "official" way of enabling it is to just issue a request to localhost... https://github.com/FrancYescO/tch_firmware_extracted/blob/vbnt-l_16.3.7190-2761003-20170907085601-501361d1f0abcd3206e49f0897c4a6cca07a114d/etc/init.d/ra

**Comment by lorenzocanalelc** [comment]
Yes, thanks for the input, because the problem it's in that service, but it's not the time, in my case the if was wrong
```
if [ ! -z $(uci show web | grep "web.state_.*\.enabled=1") 
```

This didn't work, but this 
```
if [ ! -z $(uci show web | grep "web.state_.*\.enabled='1'") 
```
Did work, because for same reason my web state file it's: 
```
web.state_remote.enabled='1'
```
Just if this is the original file I don't understand how it can work on other devices...

**Comment by lorenzocanalelc** [comment]
I Just checked and it seems that also on my DGA4132 AGTHP 2.2.1 I have the same problem but I probably didn't noticed it because I always access at least once on LAN... Did the original ra script work for you?

**Comment by FrancYescO** [comment]
this if is totally broken and also nosense as from what i can see there are no "web.state_" paths, problably these where here is some old firmware and they never fixed the script

problably we should just replace it with something like `$(uci get web.remote.active) = "1"`

**Comment by lorenzocanalelc** [comment]
I disagree with you on this, because the rest of the If is valid when remote access is enable, this how the fixed grep looks on my device with the line fixed

```
root@modemtim:~# uci show web | grep "web.state_.*\.enabled='1'"
web.state_remote.enabled='1'
```

**Comment by FrancYescO** [comment]
```
root@DGA4130:~# uci get web.state_remote.enabled
uci: Entry not found
```

**Comment by lorenzocanalelc** [comment]
Did you enable remote access before? Because the entry for me appeared only after enabling it on the GUI

**Comment by FrancYescO** [comment]
yep actually i have it enabled

**Comment by FrancYescO** [comment]
probably it depends on firmware... do you have the `uci show web.remote.active`?

**Comment by lorenzocanalelc** [comment]
I don’t know then why it is there on my DGA4132 and on the Tg789vac v2 (Both with a clean configuration) anyway, your fix should also work because I have that entry too also on mine

**Comment by lorenzocanalelc** [comment]
![DBC167C5-8486-49FF-A46D-15F0C72CC30E](https://user-images.githubusercontent.com/44505255/102722448-70b6f980-4301-11eb-922a-c6da2c4213a3.png)
As said
